scripts/caclmgrd

```diff
@@ -54,8 +54,10 @@ def get_ipv4_networks_from_interface_table(table, intf_name):
             iface_name, iface_cidr = key
             if iface_name.startswith(intf_name):
                 ip_ntwrk = ipaddress.ip_network(iface_cidr, strict=False)
-                if isinstance(ip_ntwrk, ipaddress.IPv4Network):
-                    addresses.append(ip_ntwrk)
+                ip_str = iface_cidr.split("/")[0]
```

Comment:
Could you check if this works if there are secondary IPs in Vlan? Like two or three different IPs in the same Vlan?

Reply:
Works as intended, added in description